src/components/ResumeContent.tsx

```
@@ -119,43 +119,141 @@ export const ResumeContent: React.FC<ResumeContentProps> = ({ data }) => {
+                        {work.name}
+                      </h3>
+                      <p className="text-lg text-blue-600 dark:text-blue-400 font-semibold mb-2">
+                        {work.position}
+                      </p>
+                      {work.location && (
+                        <p className="text-sm text-gray-500 dark:text-gray-400 flex items-center gap-1">
+                          <svg
+                            className="w-4 h-4"
+                            fill="currentColor"
+                            viewBox="0 0 20 20"
+                          >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)